content/release-notes/2023-09-12-storage-and-compute.md

```diff
@@ -0,0 +1,25 @@
+### Postgres extension updates
+
+The following Postgres extensions were updated to a newer version:
+
+| Postgres extension       | New version   |
```

Comment:
What do you think about adding a column with old versions?

Reply:
Good idea! I will add that.